yarn.lock

```diff
@@ -0,0 +1,1083 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please either remove the yarn.lock or the package-lock. We don't need both.

Reply:
Actioned.